azext_iot/digitaltwins/providers/resource.py

```diff
@@ -71,6 +72,25 @@ def create(
                 long_running_operation_timeout=timeout,
             )
 
+            def check_state(lro):
+                from time import sleep
+                instance = lro.resource().as_dict()
+                state = instance.get('provisioning_state', None)
+                retries = 0
+                while state == "Provisioning" and retries < MAX_ADT_CREATE_RETRIES:
+                    retries += 1
+                    sleep(1)
```

Comment:
We should wait longer than 1 second, IIRC the 'Retry-After' header value (for which the client should poll the location header URI) for this api version is 10 (seconds).

Reply:
The reason for the 1 second was through ad-hoc testing with Ryan - it seemed like some instances needed as little as 2 seconds, some up to 4 seconds, with most averaging at 3-4 seconds to propogate the hostname.

Will update to use the retry-after value to align with CLI documentation unless there is merit to keep intervals of 1 second (which I see - at least 6 seconds faster if using 1 second intervals).

Reply:
Have discussed this - will keep 10 seconds